code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -230,6 +230,17 @@
 		#ifdef DATALOGGER
 						game_stats.Increment("farts")
 		#endif
+				if(src.mutantrace && src.mutantrace.name == "dwarf" && prob(3))
+					var/list/glowsticktypes = list()
+					for(var/type in typesof(/obj/item/device/light/glowstick))
+						glowsticktypes += type
+					var/glowsticktype = pick(glowsticktypes)
+					var/obj/item/device/light/glowstick/G = new glowsticktype
```

Comment:
```suggestion
					var/obj/item/device/light/glowstick/G = new pick(typesof(/obj/item/device/light/glowstick))
```

Reply:
There was a bit of an issue with the compiler, but it's fixed in the latest commit :heart: :tada: